Sources/starlight-swift/GSet.swift

```diff
@@ -0,0 +1,31 @@
+open class GSet<T>: CRDTSet<T> where T: Hashable {
+
+    var data: Set<T>
```

Comment:
We have the public dataset var that returns data, shouldn't this be private to guarantee we won't have deletions on the G-Set data?